### PR TITLE
Designed course search result page

### PR DESCRIPTION
**Pull request by SreenathES**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import Home from "./pages/Home";
 import { Routes, Route } from "react-router-dom";
+import CourseSearch from "./pages/student/CourseSearch";
 
 function App() {
   return (
     <Routes>
-      <Route path="/" element={<Home />} />
+      <Route exact path="/" element={<Home />} />
+      <Route path="/course-search/:name" element={<CourseSearch />} />
```
Comment (mishelshaji) [inline client/src/App.js]: Why not change the URL to simply search? Also, if possible, change the name as a query parameter.

File: client/src/components/SearchBox.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export default function SearchBox() {
+    return (
+        <div className="d-flex justify-content-center align-items-center mt-3 mx-auto" id="search-box">
+            <input type='search' className='form-input p-3 is-valid' placeholder="Search For..." />
```
Comment (mishelshaji) [inline client/src/components/SearchBox.js]: Fix inconsistent use of single and double quotes for attribute values.

File: client/src/pages/Home.js [hunk 3]
```diff
@@ -63,26 +66,39 @@ export default function Home() {
         );
     }
 
+    let navigate = useNavigate();
+    
     function searchCourse() {
-        if (searchKey.length !== 0) {
-            // action
-        }
+        // Write real code here.
```
Comment (mishelshaji) [inline client/src/pages/Home.js]: Remove the comment.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 1 above)
Comment (SreenathES) [reply client/src/App.js]: resolved issues

File: client/src/components/SearchBox.js (hunk 2 above)
Comment (SreenathES) [reply client/src/components/SearchBox.js]: solved

File: client/src/pages/Home.js (hunk 3 above)
Comment (SreenathES) [reply client/src/pages/Home.js]: comment removed

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**